apps/linkedin-to-notion/src/contents/scrapers/linkedin-data-cleaners.ts

```diff
@@ -22,5 +22,5 @@ export const splitName = (fullName: string): { firstName?: string; lastName?: st
  * @returns a clean job title
  */
 export const cleanJobTitle = (jobTitle: string): string => {
-  return jobTitle.split(/ at| @| chez/g)[0].trim();
+  return jobTitle.split(/ at| @| chez/g)[0]?.trim() || jobTitle;
```

Comment:
I don't understand why the "or" condition is necessary. If there's no separator in the string, it will still return something coherent. For instance:
```js
"Growth Engineer".split('|')[0]?.trim()
// => "Growth Engineer"
```
Is it to remove typing errors again?

Reply:
It is and it's necessary. Split can return undefined => [0] of undefined is an error => your app crashes 😬